### DISCUSSION TITLE
Title component is not taking globally configured font

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [mantinedev/mantine#1746] [Q&A] [answered]
Hi. I am using globally configured font and every component's font that i am using is OK except Title component. please help. thanks.

Edit:
It is working with `sx={{fontFamily: 'unset'}}`.

**Comment by rtivital** [comment, marked as answer]
You can change Title component font-family on theme – https://mantine.dev/theming/extend-theme/#typography

**Reply by MadadiSaeed** [reply]
Thanks for the response. It is working now.
